src/sqlfluff/rules/L036.py

```diff
@@ -301,12 +315,7 @@ def _fixes_for_move_after_select_clause(
                         ]
                         fixes_.append(
                             LintFix.create_after(
-                                self._choose_anchor_segment(
-                                    context,
-                                    "create_after",
-                                    select_clause[0],
-                                    filter_meta=True,
-                                ),
+                                select_clause[0],
```

Comment:
Unrelated to the issue, but no longer necessary to call `_choose_anchor_segment()` here due to recent improvements in the core linter.